versions.cfg

```diff
@@ -43,4 +43,11 @@ repoze.who = 2.2
 RestrictedPython = 3.6.0
 zope.sequencesort = 4.0.1
 zope.testbrowser = 3.11.1
+ZODB = 4.2.0
+sphinx-rtd-theme = 0.1.9
+alabaster = 0.7.7
+Babel = 2.2.0
+snowballstemmer = 1.2.1
+
+# only for buildout dependencies
 ZODB3 = 3.11.0
```

Comment:
This is pulled in by the following dependencies:
https://github.com/zopefoundation/Products.BTreeFolder2/
https://github.com/zopefoundation/Products.ExternalMethod/
https://github.com/zopefoundation/Products.ZCatalog/
https://github.com/zopefoundation/Products.ZCTextIndex/

I propose to remove the explicit dependency on ZODB (3) in these packages because they all depend on Zope2 anyways.